staging/src/k8s.io/client-go/discovery/cached/disk/round_tripper.go

```diff
@@ -18,15 +18,14 @@ package disk
 
 import (
 	"net/http"
-	"os"
-	"path/filepath"
 
 	"github.com/gregjones/httpcache"
-	"github.com/gregjones/httpcache/diskcache"
-	"github.com/peterbourgon/diskv"
 	"k8s.io/klog/v2"
 )
 
+// 100MB suggested disk cache size; this is NOT a hard limit.
+const cacheSize int64 = 100 * 1024 * 1024
```

Comment:
Cached files in `.kube/cache/http` take roughly ~3KB in average, that means ~35K files can be stored actively which looks reasonable to me(at least thanks to that PR, we have upper limit now). Maybe in the future, we can provide an option to users to increase or decrease that value according to their needs.

Reply:
I'm thinking of making the 100MB disk cache size larger, and I'd love to hear feedback on that. As far as making the value dynamic and user-settable, I think that would be very useful too. I'm hoping to address that in a future PR.